consensus/replay.go

```diff
@@ -324,7 +324,10 @@ func (h *Handshaker) ReplayBlocks(
 			}
 
 			if res.ConsensusParams != nil {
+				timeIotaMs := state.ConsensusParams.Block.TimeIotaMs
 				state.ConsensusParams = types.PB2TM.ConsensusParams(res.ConsensusParams)
+				// TimeIotaMs is not a part of ABCI consensus params
+				state.ConsensusParams.Block.TimeIotaMs = timeIotaMs
```

Comment:
I did not verify that this is the only place where we do not want ABCI params to potentially overwrite `TimeIotaMs`.  Was this caught by a test?

Reply:
No, me executing

```
 rg "PB2TM.ConsensusParams"
consensus/replay.go
328:                            state.ConsensusParams = types.PB2TM.ConsensusParams(res.ConsensusParams)

types/protobuf_test.go
67:     cp2 := PB2TM.ConsensusParams(abciCP)
```